git-format-pkg-patch

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 #
 # Copyright (c) 2017, SUSE Linux Products GmbH
```

Comment:
I think we should update the copyright attribution:
```suggestion
# Copyright (c) 2017-2020, SUSE LLC
```

Reply:
👍